templates/tfengine/components/cicd/configs/tf-apply.yaml

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Note: Terraform version used in the automation is 0.12.29.
+{{- $cft := "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:47db3e958fbaa0d95881cf99501ebf4522a261ddc68d9566b7cf70e26cf7cddb"}}
```

Comment:
Change the rest of the file to use this variable.

Reply:
good catch